packages/unenv-preset/src/preset.ts

```diff
@@ -112,3 +113,59 @@ export function getCloudflarePreset({
 		external: nodeCompatModules.flatMap((p) => [p, `node:${p}`]),
 	};
 }
+
+/**
+ * Returns the aliases for node http modules (unenv or workerd)
+ *
+ * The native implementation:
+ * - is enabled after 2025-08-15
+ * - can be enabled with the "enable_nodejs_http_modules" flag
+ * - cab be disabled with the "disable_nodejs_http_modules" flag
+ */
+function getHttpAliases({
+	compatibilityDate,
+	compatibilityFlags,
+}: {
+	compatibilityDate: string;
+	compatibilityFlags: string[];
+}): Record<string, string> {
+	const disabledByFlag = compatibilityFlags.includes(
+		"disable_nodejs_http_modules"
+	);
+	const enabledByFlags = compatibilityFlags.includes(
+		"enable_nodejs_http_modules"
+	);
+	const enabledByDate = compatibilityDate >= "2025-08-15";
+
+	const enabled = (enabledByFlags || enabledByDate) && !disabledByFlag;
+
+	if (!enabled) {
+		// use the unenv polyfill
+		return {};
+	}
+
+	const aliases: Record<string, string> = {};
+
+	const nativeModules = [
+		"_http_common",
+		"_http_outgoing",
+		"_http_client",
+		"_http_incoming",
+		"_http_agent",
+	];
+
+	for (const nativeModule of nativeModules) {
+		aliases[nativeModule] = nativeModule;
```

Comment:
Do we need this? It is saying that `aliases["_http_common"] = "_http_common";` isn't that a no-op?

Reply:
Or is this overriding aliases from the standard preset?